AndelaOSP/Andela-Socials
# #166497823 Delete calendar event

#### What Does This PR Do?
It deletes the event from all attendee calendar when the host deletes the event
#### Description Of Task To Be Completed
- Delete event from the calendar using the google calendar API
- Write tests for the implementation
#### Any Background Context You Want To Provide?
None
#### How can this be manually tested?
- Fetch the branch locally by running `git fetch origin ft-delete-calendar-event-166497823` & `git checkout ft-delete-calendar-event-166497823` respectively
- Create an event whose category matches with the interest of another user
- Delete the event. The event should not be in your calendar and the calendar of the attendees

#### What are the relevant pivotal tracker stories?
[166497823](https://www.pivotaltracker.com/story/show/166497823)
#### Screenshot(s)
None
